lib/publiccloud/aws_client.pm

```diff
@@ -98,6 +100,26 @@ sub init {
     die("The UserID doesn't have the correct format: $self->{user_id}") unless $self->aws_account_id =~ /^\d{12}$/;
 }
 
+=head2 get_container_registry_prefix
+Get the full registry prefix URL (based on the account and region) to push container images on ECR.
+=cut
+sub get_container_registry_prefix {
+    my ($self) = @_;
+    my $region = $self->region;
+    my $full_name_prefix = sprintf('%s.dkr.ecr.%s.amazonaws.com', $self->aws_account_id, $region);
+    return $full_name_prefix;
+}
+
+=head2 get_container_image_full_name
+Get the full name for a container image in ECR registry
```

Comment:
The `$tag` parameter is missing as possible argument

Reply:
I'm thinking about something like adding

    C<tag> Tag of the container to ge t the full name of

Or something similar, analog to e.g. https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/containers/engine.pm#L56

Reply:
Done